compiler/resolution/initializerResolution.cpp

```diff
@@ -116,16 +116,20 @@ resolveInitializer(CallExpr* call, bool emitCallResolutionErrors) {
   return retval;
 }
 
-static std::map<FnSymbol*,FnSymbol*> newWrapperMap;
+static std::map<std::pair<FnSymbol*,Expr*>,FnSymbol*> newWrapperMap;
```

Comment:
Please add a comment describing the meaning of this map -- what are the key's tuple components, and what is the value?